app/app/explore/TopCard.tsx

```diff
@@ -19,31 +24,43 @@ export const TopCard: React.FC<TopCardProps> = ({
   getAssetSymbol,
   getLogoPath,
   children,
+  index,
 }) => {
+  const innerCardClass =
+    "bg-white rounded-xl md:p-3 p-1 flex justify-center flex-col items-center flex-1 w-1/2";
+  const valueClass = twMerge(
+    "md:text-2xl font-medium",
+    index! > 0 ? "text-base" : "text-xl",
+  );
+  const labelClass = "text-gray-700 md:text-base text-xs mt-1 font-medium";
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    setIsMobile(window.innerWidth < 768);
```

Comment:
not a fan of having to `use client` just for this. there has to be a better way to adjust the image size to make it responsive.